fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectoryLockFactory.java

```diff
@@ -57,7 +57,7 @@ public Lock obtainLock(final Directory dir, final String lockName) {
         return new FDBDirectoryLock(directory.getAgilityContext(), lockName, directory.fileLockKey(lockName), timeWindowMilliseconds);
     }
 
-    private static class FDBDirectoryLock extends Lock {
+    protected static class FDBDirectoryLock extends Lock {
```

Comment:
Spotbugs failed because you are now exposing a bunch of internal state.
Changing all the fields, and the constructor to `private` seems to fix the issue. I would do it as a suggested change, but I can't because you didn't change those lines.

Reply:
Looks good to me.